### PR TITLE
Use date instead of time duration

### PR DESCRIPTION
**Pull request by Swechhya**
## Description
- We are currently using duration and calculating date. Use start date instead
- Update github action to run every midnight

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -15,16 +13,19 @@
...
-    "timedelta": get_date(duration),
+    "timedelta": start_date,
     "dataCount": data_count,
 }
 
 
 def main():
     """Script entrypoint."""
 
+    print(variables)
```
Comment (bhattaraib58) [inline main.py:28]: Do we need to print out variables? or is it missed?

Comment (Swechhya) [reply main.py:28]: Missed, will change it

**Review by mesaugat — APPROVED** [review]
Looks good. @Swechhya Do we know why the netlify preview URL isn't working?

**Comment by Swechhya** [comment]
> Looks good. @Swechhya Do we know why the netlify preview URL isn't working?

@mesaugat  We haven't added user list yet. After we add that it should run successfully.

**Comment by mesaugat** [comment]
@Swechhya Okay. So as soon as the contribution begins we should get the users right?

**Pull request merged**